src/test/scala/nl/knaw/dans/easy/solr/ConfSpec.scala

```diff
@@ -0,0 +1,107 @@
+package nl.knaw.dans.easy.solr
+
+import java.io.{ByteArrayOutputStream, File}
+import java.lang.System.{clearProperty, setProperty}
+import java.net.MalformedURLException
+
+import org.apache.commons.configuration.PropertiesConfiguration
+import org.apache.commons.io.FileUtils.{readFileToString, write}
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.collection.JavaConverters._
+import scala.runtime.BoxedUnit
+
+class ConfSpec extends FlatSpec with Matchers {
+
+  // by default tests don't use any cfg/application.properties
+  clearProperty("app.home")
+
+  "options in help info" should "be part of README.md" in {
+    // clearProperty("app.homedir") does not help a maven build so this test should come first
+    val readme = readFileToString(new File("README.md"))
+    val options = printHelp().split("Options:")(1)
+    if (!trim(readme).contains(trim(options)))
+      throw new RuntimeException(s"README.md should contain:\n$options")
+  }
+
+  it should "show all items in distributed application.properties" in {
+
+    write(new File("target/test/cfg/application.properties"),
+    """default.fcrepo-server=http://127.0.0.1:8080/fedora
+      |default.fcrepo-user=somebody
+      |default.fcrepo-password=secret
+      |default.solr-update-url=http://127.0.0.1:8080/solr
+      |default.dataset-batch-size=10
+      |default.dataset-timeout=100
+      | """.stripMargin)
+
+    val testProps = new PropertiesConfiguration("target/test/cfg/application.properties")
+    val distProps = new PropertiesConfiguration("src/main/assembly/dist/cfg/application.properties")
+    sortedKeys(distProps) should be (sortedKeys(testProps))
+
+    setProperty("app.home", "target/test")
+    val helpInfo = printHelp().replaceAll("\n", " ").replaceAll(" +"," ")
+
+    for (key <- testProps.getKeys.asScala)
+      helpInfo should include(s"(default = ${testProps.getString(key)})")
+
+    helpInfo should include("target/test/cfg/application.properties") // "defaults provided by"
+  }
+
+  "version variable" should "rather be substituted" in {
+    printHelp() should include("${project.version}")
+    // FIXME
```

Comment:
During the maven build's resource filtering. I don't think this test is really necessary.